app/src/main/res/layout/activity_newsletter.xml

```diff
@@ -1,13 +1,138 @@
+            android:gravity="center_vertical"
+            android:orientation="horizontal">
+
+            <TextView
+              android:id="@+id/subscribe_all_text_view"
+              style="@style/NewsletterSingleRow"
+              android:text="@string/profile_settings_newsletter_subscribe_all" />
+
+            <android.support.v7.widget.SwitchCompat
+              android:id="@+id/subscribe_all_switch"
+              style="@style/NewsLetterSwitch" />
+
+          </LinearLayout>
+        </LinearLayout>
+
+        <LinearLayout
+          style="@style/SettingsLinearRow"
+          android:layout_marginTop="@dimen/grid_5">
+
+          <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/grid_10"
+            android:gravity="center_vertical"
+            android:orientation="horizontal">
+
+            <TextView
+              android:id="@+id/projects_we_love_text_view"
+              style="@style/NewsletterSingleRow"
+              android:text="@string/Projects_We_Love" />
+
+            <android.support.v7.widget.SwitchCompat
+              android:id="@+id/projects_we_love_switch"
+              style="@style/NewsLetterSwitch" />
+
+          </LinearLayout>
+        </LinearLayout>
+
+        <TextView
+          style="@style/NewsLetterTextView"
+          android:text="@string/Sign_up_newsletter" />
+
+        <LinearLayout
+          style="@style/SettingsLinearRow">
+
+          <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/grid_10"
+            android:gravity="center_vertical"
+            android:orientation="horizontal">
+
+            <TextView
+              style="@style/NewsletterSingleRow"
+              android:text="@string/profile_settings_newsletter_happening" />
+
+            <android.support.v7.widget.SwitchCompat
+              android:id="@+id/happening_switch"
+              style="@style/NewsLetterSwitch" />
+
+          </LinearLayout>
+        </LinearLayout>
+
+        <TextView
+          style="@style/NewsLetterTextView"
+          android:text="@string/Happening_newsletter" />
+
+        <LinearLayout
+          style="@style/SettingsLinearRow">
+
+          <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/grid_10"
+            android:gravity="center_vertical"
+            android:orientation="horizontal">
+
+            <TextView
+              style="@style/NewsletterSingleRow"
+              android:text="@string/profile_settings_newsletter_news_event" />
+
+            <android.support.v7.widget.SwitchCompat
+              android:id="@+id/news_events_switch"
+              style="@style/NewsLetterSwitch" />
+
+          </LinearLayout>
+        </LinearLayout>
+
+        <TextView
+          style="@style/NewsLetterTextView"
+          android:text="@string/News_events" />
+
```

Comment:
I think we're missing the Games newsletter.

Reply:
See comment below.